gammapy/background/energy_offset_array.py

```diff
@@ -104,3 +127,76 @@ def plot_image(self, ax=None, offset=None, energy=None, **kwargs):
         image = ax.imshow(self.data, extent=extent, **kwargs)
         plt.colorbar(image)
         return ax
+
+    @classmethod
+    def read(cls, filename):
+        """Create `EnergyOffsetArray` from  fits file.
+
+        Parameters
+        ----------
+        filename : str
+            File name
+        """
+        hdu_list = fits.open(filename)
+        return cls.from_fits(hdu_list)
+
+    
+    @classmethod
+    def from_fits(cls, hdu, scheme=None):
+        """Read EnergyOffsetArray from a fits binary table.
+
+        Parameters
+        ----------
+        hdu : `~astropy.io.fits.BinTableHDU`
+            HDU binary table for the EnergyOffsetArray.
```

Comment:
The formatting for this docstring doesn't come out OK in HTML:
![screen shot 2016-02-10 at 09 12 15](https://cloud.githubusercontent.com/assets/852409/12941825/76f533f0-cfd6-11e5-9009-84f95fe7a462.png)

I think you have to add an empty line before the line with "Returns" and then it'll be OK.